scripts/validation/embed-logs/Message Object-json-format.json

```diff
@@ -0,0 +1,730 @@
+          }
+        ]
+      }
+    },
+    "serverVariables": {
+      "stage": {
+        "default": "demo",
+        "description": "This value is assigned by the service provider in this example of `mycompany.com`"
+      }
+    },
+    "channels": {
+      "user": {
+        "address": "users.{userId}",
+        "title": "Users channel",
+        "description": "This channel is used to exchange messages about user events.",
+        "messages": {
+          "userSignedUp": {
+            "$ref": "#/components/messages/userSignUp"
+          }
+        },
+        "parameters": {
+          "userId": {
+            "$ref": "#/components/parameters/userId"
+          }
+        },
+        "servers": [
+          {
+            "$ref": "#/components/servers/development"
+          }
+        ],
+        "bindings": {
+          "$ref": "#/components/channelBindings/user"
+        },
+        "tags": [
+          {
+            "$ref": "#/components/tags/user"
+          }
+        ],
+        "externalDocs": {
+          "$ref": "#/components/externalDocs/infoDocs"
+        }
+      }
+    },
+    "messages": {
+      "userSignUp": {
+        "summary": "Action to sign a user up.",
+        "traits": [
+          {
+            "$ref": "#/components/messageTraits/commonHeaders"
+          }
+        ],
+        "payload": {
+          "$ref": "#/components/schemas/Category"
+        },
+        "correlationId": {
+          "$ref": "#/components/correlationIds/default"
+        },
+        "bindings": {
+          "$ref": "#/components/messageBindings/user"
+        },
+        "examples": []
+      },
+      "userSignedUp": {
+        "name": "UserSignup",
+        "title": "User signup",
+        "summary": "Action to sign a user up.",
+        "description": "A longer description",
+        "contentType": "application/json",
+        "tags": [
+          {
+            "name": "user"
+          },
+          {
+            "name": "signup"
+          },
+          {
+            "name": "register"
+          }
+        ],
+        "headers": {
+          "type": "object",
+          "properties": {
+            "correlationId": {
+              "description": "Correlation ID set by application",
+              "type": "string"
+            },
+            "applicationInstanceId": {
+              "description": "Unique identifier for a given instance of the publishing application",
+              "type": "string"
+            }
+          }
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "user": {
+              "$ref": "#/components/schemas/userCreate"
+            },
+            "signup": {
+              "$ref": "#/components/schemas/signup"
+            }
+          },
+          "schemaFormat": "application/vnd.apache.avro+json;version=1.9.0",
+          "schema": {
+            "$ref": "path/to/user-create.avsc#/UserCreate"
```

Comment:
The error message for this line:
```bash
Warning in Message Object-json-format: ENOENT: no such file or directory, open 'path/to/user-create.avsc'
```
Same error for the yaml version as well.
@smoya potential fix?

Reply:
The error is because the file `path/to/user-create.avsc` is expected to exist.
You have two options here:

1. To include an avro file (`user-create.avsc`) in a known path like `.github/workflows/files` or similar path (not having a good suggestion here tbh)
2. Not so ideal, but to exclude the example from testing

Reply:
I think including it would be much better. Let me work on it... :eyes:

Reply:
Validating perfectly after the changes made via f814d82